tool/utils.ts

```diff
@@ -155,6 +157,21 @@ export async function getDartSassEmbedded(
   await link(p.join(source, 'build'), p.join(outPath, repo));
 }
 
+/**
+ * Throws an informative error if we're running in a Linux environment that uses
+ * musl.
+ */
+async function checkForMusl(): Promise<void> {
+  if (process.platform !== 'linux') return;
+
+  const executable = await fs.readFile(process.execPath);
+  if (!executable.includes('libc.musl-x86_64.so.1')) return;
```

Comment:
We should also consider aarch64 as well as other arch here.

Reply:
I'm not intimately familiar with what strings those are likely to leave in the executable. Can you give me a list of strings to search for?

Reply:
https://pkgs.alpinelinux.org/contents?file=&path=&name=musl&branch=edge&repo=main

Reply:
Would it make sense to check against `'/lib/ld-musl-'` and `'/lib/libc.musl-'` so it works regardless of architecture?

Reply:
It is extremely rare but multiple ld/libc can be installed in the same system at non-standard locations, so searching with `/lib/` might miss some rare cases. If we want good performance in searching, `libc.musl-` is probably the best.